srcpkgs/didder/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'didder'
+pkgname=didder
+version=1.1.0
+revision=1
+build_style=go
+go_import_path="github.com/makeworld-the-better-one/didder"
+go_ldflags="-s -w -X main.version=${version} -X main.builtBy=Void"
+hostmakedepends="pandoc"
+short_desc="Command-line image dithering tool"
+maintainer="Dakota Walsh <kota@nilsu.org>"
+license="GPL-3.0-or-later"
```

Comment:
```suggestion
license="GPL-3.0-only"
```
does not mention the "or any later version" language in the repo

Reply:
Changed! I always thought maybe it was "or-later" unless specified otherwise...